safe-global/safe-wallet-web
# fix: Update illustration for new tx

## What it solves

Part of #2067 

## How this PR fixes it

- Updates the illustration when creating a new transaction

## How to test it

1. Open a Safe
2. Press New transaction
3. Observe the new image
4. Switch to dark/light mode
5. Observe the image changing

## Screenshots

<img width="1109" alt="Screenshot 2023-07-07 at 09 41 42" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/cb19af75-530c-40e5-a533-24e95705a824">

<img width="1106" alt="Screenshot 2023-07-07 at 09 41 50" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/fcca75a6-a628-41bb-9be0-cc8e19ddd3d4">

## Checklist
